### DISCUSSION TITLE
Font change

### DISCUSSION THREAD

**Discussion by Blobst** [nix-community/nix-on-droid#364] [General]
See I want to change the font to JetBrains mono but everytime I search for something nothing shows up so please help 

Nix-on-droid bootstrap aarch64 23.11

**Comment by t184256** [comment]
Define "everytime I search for something nothing shows up".

**Comment by Blobst** [comment]
Like I mean I don't see any search results of people changing the font on nix-on-droid

**Comment by t184256** [comment]
Well, there's an [option](https://nix-community.github.io/nix-on-droid/nix-on-droid-options.html#opt-terminal.font) for changing the font, implemented in a [PR](https://github.com/nix-community/nix-on-droid/pull/201/files) filed as a response to an [issue](https://github.com/nix-community/nix-on-droid/issues/120).

**Comment by Readf0x** [comment]
Simply place your font at `~/.termux/font.ttf`, as stated in the termux documentation.

**Reply by voronind-com** [reply]
Also note that home.file won't work as Termux will fail to read file before nix store symlink would be available.

I instead copy it on switch with my Makefile.

**Reply by t184256** [reply]
Congrats, you've reinvented what [the option](https://nix-community.github.io/nix-on-droid/nix-on-droid-options.html#opt-terminal.font) does.

**Reply by voronind-com** [reply]
> Congrats, you've reinvented what [the option](https://nix-community.github.io/nix-on-droid/nix-on-droid-options.html#opt-terminal.font) does.

Yay! I'll switch to them, thanks! In my defense, I don't remember them existing when I started using NoD.
By the way, no plans on creating a chat somewhere?
Also would that be a good idea to write in wiki how I added Stylix to NoD? It has some quirks compared to standalone HM configuration. You can see screenshots here https://git.voronind.com/voronind/nix#screenshots

Edit: darh, I rotate screenshots over time, last two don't really have enough difference in them. But that allows NoD to follow desktop's colors in CLI tools.

Edit2: Thank you very much for NoD. I enjoy it enormously.

**Comment by t184256** [comment]
There's a matrix chatroom, not very active. Not familiar with Stylix, sorry.

**Reply by voronind-com** [reply]
FYI https://github.com/danth/stylix
